### ISSUE TITLE
[master] Stock settings traceback

### ISSUE THREAD

**Issue by nhomar** [odoo/odoo#19502]
Impacted versions:

master at today

Steps to reproduce:

Try set the configuration for inventory

Current behavior:

traceback:

```
Error:
Uncaught TypeError: widget.renderWithLabel is not a function

http://265097-master-270c01.runbot14.odoo.com/web/content/2078-0fc82b6/web.assets_backend.js:1337
Traceback:
TypeError: widget.renderWithLabel is not a function
    at http://265097-master-270c01.runbot14.odoo.com/web/content/2078-0fc82b6/web.assets_backend.js:1337:52
    at Function._.each._.forEach (http://265097-master-270c01.runbot14.odoo.com/web/content/2071-e38fe79/web.assets_common.js:12:558)
    at Class._updateView [as _super] (http://265097-master-270c01.runbot14.odoo.com/web/content/2078-0fc82b6/web.assets_backend.js:1336:157)
    at Class._updateView (http://265097-master-270c01.runbot14.odoo.com/web/content/2078-0fc82b6/web.assets_backend.js:2984:20)
    at Class._updateView (http://265097-master-270c01.runbot14.odoo.com/web/content/2071-e38fe79/web.assets_common.js:3813:371)
    at http://265097-master-270c01.runbot14.odoo.com/web/content/2078-0fc82b6/web.assets_backend.js:1335:1479
    at http://265097-master-270c01.runbot14.odoo.com/web/content/2071-e38fe79/web.assets_common.js:802:681
    at fire (http://265097-master-270c01.runbot14.odoo.com/web/content/2071-e38fe79/web.assets_common.js:796:299)
    at Object.add [as done] (http://265097-master-270c01.runbot14.odoo.com/web/content/2071-e38fe79/web.assets_common.js:797:467)
    at Array.<anonymous> (http://265097-master-270c01.runbot14.odoo.com/web/content/2071-e38fe79/web.assets_common.js:802:649)
```

Expected behavior:

No traceback

Video/Screenshot link (optional):

**Comment by nhomar** [comment]
saas-17 also.

**Comment by sle-odoo** [comment]
@mge-odoo not tested, could be linked to cb540c48eb575902018da97f2f541a813bda4d6b

**Comment by mge-odoo** [comment]
@sle-odoo yep, indeed, my bad. @aab-odoo is working on a fix right now

**Comment by sle-odoo** [comment]
fixed by rev ae14f93464ac9878180b50f262a84804538fb6f3 in enterprise
